bootstrap.sh

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env bash
 # Usage: ./bootstrap.sh
-# This script will install swiftformat, link git hooks and install required ruby gems.
+# This script will install Mint and bootstrap its dependencies, link git hooks and install required ruby gems.
 # You should have homebrew installed.
 # If you get `zsh: permission denied: ./bootstrap.sh` error, please run `chmod +x bootstrap.sh` first
 
-# Install swiftformat 0.45.6
-brew unlink swiftformat
-brew tap-new getstream/ios
-brew extract --version 0.45.6 swiftformat getstream/ios
-brew reinstall getstream/ios/swiftformat@0.45.6
-brew untap getstream/ios
+SWIFTFORMAT_VERSION=0.45.6
+SWIFTLINT_VERSION=0.41.0 # Also used by danger-swiftlint via bundle config
+
+# Check if Homebrew is installed
+if [[ $(command -v brew) == "" ]]; then
+    echo "Homebrew not installed. Please install."
+    exit 1
+fi
+
+echo
+echo "👉 Generate mintfile"
+echo "# Warning - this file is generated using bootstrap.sh, don't edit it directly." > mintfile
+echo "nicklockwood/SwiftFormat@$SWIFTFORMAT_VERSION" >> mintfile
+echo "realm/SwiftLint@$SWIFTLINT_VERSION" >> mintfile
+cat mintfile
+
+echo
+echo -e "👉 Install Mint if needed"
+# List installed Mint versions, if fails, install Mint
+brew list mint || brew install mint
 
 # Set bash to Strict Mode (http://redsymbol.net/articles/unofficial-bash-strict-mode/)
 set -euo pipefail
 
+echo
+echo -e "👉 Bootstrap Mint dependencies"
+mint bootstrap
+
 # Symlink hooks folder to .git/hooks folder
+echo
+echo -e "👉 Create symlink for pre-commit hooks"
 ln -sf ../../hooks/pre-commit.sh .git/hooks/pre-commit
```

Comment:
Am I missing something or why `../../hooks` and not `./hooks`?

Reply:
I have no idea 🤷‍♂️  It seems everything is working correctly with `./hooks`. We even use it on the following line too. Fixed in 08a0e2d